src/libraries/Microsoft.Extensions.Logging.EventSource/src/LoggingEventSource.cs

```diff
@@ -338,14 +341,21 @@ private static LoggerFilterRule[] ParseFilterSpec(string filterSpec, LogLevel de
 
             var rules = new List<LoggerFilterRule>();
 
-            // All event source loggers are disabled by default
-            rules.Add(new LoggerFilterRule(typeof(EventSourceLoggerProvider).FullName, null, LogLevel.None, null));
+            if (filterSpec == null || !filterSpec.StartsWith(UseAppFilters, StringComparison.OrdinalIgnoreCase))
```

Comment:
Obscure corner case: FilterSpecs=UseAppFiltersIAmAJerk:Debug;Component1:Info

If we want to be accurate lets split on semicolon first, then confirm the the first entry equals "UseAppFilters" instead of only starting with it.